src/index.html

```diff
@@ -1,13 +1,360 @@
+    <!--#Aside  -->
+
+    <!-- Events -->
+    <section class="events" id="events">
+      <div class="container">
+        <h2 class="events-title title">Актуальні події</h2>
+        <div class="events-cards">
+          <div class="events-card">
+            <img
+              class="events-card__image"
+              src="./images/events-card-1.png"
+              alt="events img"
+            >
+            <div class="events-card__content">
+              <div class="events-card__info">
+                <p class="events-card__sutitle">ВИСТАВКА</p>
+                <p class="events-card__date">26.08-29.11.2019</p>
+              </div>
+              <div class="events-card__title-container">
+                <h2 class="events-card__title">Йду і повертаюсь</h2>
+                <div class="events-card__title-mark"></div>
+              </div>
+              <p class="events-card__desc">
+                Національний Художній Музей України презентує унікальну частину колекції Градобанку - українське мистецтво другої половини 1980-1995 років.
+              </p>
+            </div>
+          </div>
+          <div class="events-card">
+            <img
+              class="events-card__image"
+              src="./images/events-card-2.png"
+              alt="events img"
+            >
+            <div class="events-card__content">
+              <div class="events-card__info">
+                <p class="events-card__sutitle">ВИСТАВКА</p>
+                <p class="events-card__date">26.08-29.11.2019</p>
+              </div>
+              <div class="events-card__title-container">
+                <h2 class="events-card__title">І спогади і мрії</h2>
+                <div class="events-card__title-mark"></div>
+              </div>
+              <p class="events-card__desc">
+                Національний художній музей України до 100 річчя від дня народження   видатної української художниці  Тетяни Яблонської  відкриває ретроспективну ювілейну виставку «І спогади і мрії».
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="events-lection">
+        <img
+          src="./images/events-card-3.png"
+          alt="events lection"
+          class="events-lection__img"
+        >
+        <div class="events-lection__box">
+          <div class="events-lection__head">
+            <h3 class="events-lection__subtitle">ЛЕКЦІЯ</h3>
+            <p class="events-lection__date">26/08/2019 - 11:00</p>
+          </div>
+          <h2 class="events-lection__title">Від класицизму до романтизму</h2>
+        </div>
+      </div>
+    </section>
+    <!-- #Events -->
+
+    <!-- Gallery -->
+    <div class="gallery" id="gallery">
+      <div class="container">
+        <div class="gallery__slides">
+          <div class="gallery__slide gallery__slide-1">
+            <img
+            src="./images/gallery-1.png"
+            alt="slide-1"
+            class="gallery__img"
+            >
```

Comment:
Need to fix it everywhere
```suggestion
            <img
              src="./images/gallery-1.png"
              alt="slide-1"
              class="gallery__img"
            >
```